x-pack/plugins/beats_management/public/components/tag/config_view/config_form.tsx

```diff
@@ -74,8 +74,8 @@ interface ComponentProps {
   values: ConfigurationBlock;
   schema: YamlConfigSchema[];
   id: string;
+  onSubmit?: (modal: any) => any;
```

Comment:
For my undestanding, what is this change around `onSubmit` doing?

Reply:
making onSubmit prop optional. I made the component so that if you don't pass in a callback for onSubmit the UI goes into read-only mode